recipes/ccphylo/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "ccphylo" %}
+{% set version = "0.6.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://bitbucket.org/genomicepidemiology/{{ name }}/get/{{ version }}.tar.gz
+  sha256: d23860f90df6d546f340d53f294070070406b4f80e7e1f3d8ecbfaaefb7ca9bf
+
+build:
+  skip: True  # [win]
+  number: 0
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+  host:
+    - zlib
+  run:
+    - zlib
+  
+test:
+  commands:
+   - ccphylo --help | grep CCPhylo >/dev/null
```

```suggestion
   - ccphylo --help
```

I think this is good enough. Or do you have a special reason to grep for the program name?